sklearn/metrics/tests/test_pairwise.py

```diff
@@ -499,6 +500,14 @@ def test_rbf_kernel():
     assert_array_almost_equal(K.flat[::6], np.ones(5))
 
 
+def test_laplacian_kernel():
```

Comment:
Would it be possible to include a test of non-diagonal elements as well? There are ways of getting a diagonal of 1 without proper computation.